lerobot/configs/robot/arx5.yaml

```diff
@@ -0,0 +1,26 @@
+_target_: lerobot.common.robot_devices.robots.arx5.ARX5Robot
+leader_arms:
+  main:
+    model: L5
+    interface_name: enx5c5310ecc0ec
```

Comment:
these need to be set based on your own robot's serial number. I wonder if there is a way to allow a user to specify them without making edits to this file?